addons/website/static/tests/builder/website_builder/carousel_item.test.js

```diff
@@ -55,3 +55,29 @@ test("Remove slide", async () => {
     expect(":iframe .carousel-indicators > *").toHaveCount(2);
     expect(":iframe .carousel-indicators > .active").toHaveCount(1);
 });
+
+async function clickButtonAndWait(selector, slideNumber) {
+    await contains(`button[title='${selector}']`).click();
+    await waitFor(`[data-container-title='Slide (${slideNumber}/${slideNumber})']`);
+}
+
+function checkButtonDisable(selector) {
+    expect(selector).toHaveClass("disabled");
+}
+
+test("Check reordering and slide navigation buttons in carousel are disabled when there is only one slide", async () => {
```

Comment:
Nit:

may be `should disable reordering and navigation buttons when carousel has a single slide`